tests/benchmarks/test_basic_parity.py

```diff
@@ -97,7 +97,6 @@ def measure_loops(cls_model, kind, num_runs=10, num_epochs=10):
         gc.collect()
         if device_type == "cuda":
             torch.cuda.empty_cache()
-            torch.cuda.reset_max_memory_cached()
```

Comment:
`reset_max_memory_cached()` only calls `reset_peak_memory_stats()` inside, which is not needed as we call the latter below.

You can check the impl. here: https://github.com/pytorch/pytorch/blob/a0ada2d22b0e612dbf50fa5e94e6454713af8bb4/torch/cuda/memory.py#L278

Reply:
Same applies to `reset_max_memory_allocated`.

https://github.com/pytorch/pytorch/blob/a0ada2d22b0e612dbf50fa5e94e6454713af8bb4/torch/cuda/memory.py#L252